### DISCUSSION TITLE
Is there a way to implicitly force next state handler?

### DISCUSSION THREAD

**Discussion by xahon** [aiogram/aiogram#904] [Q&A] [answered]
How to trigger `@dp.message_handler` from previous one without "asking" user to send a message by themselves. I don't want to extract my handler body into a function that will be called in previous handler and actual state handler

```
@dp.message_handler(state=FirstStateGroup.some_state)
async def process_firstStateGroup_someState(message: types.Message, state: FSMContext):
    await bot.send_message("You are in FirstStateGroup.some_state")
    await bot.send_message("Now you will be redirected to another state")
    await SecondStateGroup.first()

@dp.message_handler(state=SecondStateGroup.some_state)
async def process_secondStateGroup(message: types.Message, state: FSMContext):
    await bot.send_message("You are in SecondStateGroup.some_state")
```

I want user to see 3 messages in a row without user's interaction at all
```
You are in FirstStateGroup.some_state
Now you will be redirected to another state
You are in SecondStateGroup.some_state
```

Theoretically i can extract `process_secondStateGroup()` body but there are downsides:
1. It can depend on state which is unknown in `process_firstStateGroup()` function
2. I just don't want to create those intermediate functions that complicate reading of the code

I need some kind of forcing next state with an empty message or something like that

**Comment by Olegt0rr** [comment, marked as answer]
If you don't want to handle a message (i.e. `without interactions`), then you don't need a handler. 
Just create function you want to call and call it!
